tests/shared/drive-constraints.spec.js

```diff
@@ -556,4 +556,229 @@ describe('Shared: DriveConstraints', function() {
 
   });
 
+  describe('.getDriveImageCompatibilityStatus', function() {
+
+    beforeEach(function() {
+      if (process.platform === 'win32') {
+        this.mountpoint = 'E:\\foo';
```

Comment:
Does this mean [this](https://github.com/resin-io/etcher/blob/ae99b8b43506c667ff5282e27d2505b3bbdad3fe/tests/shared/drive-constraints.spec.js#L468) is wrong too? Should I fix it?